features/record-detail/components/swim-stats-item.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@/styled-system/css';
+import { flex } from '@/styled-system/patterns';
+
+export const SwimStatsItem = () => {
+  return (
+    <div className={containerStyle}>
+      <h1 className={text.type}>자유형</h1>
+      <h2 className={text.distance}>1500m</h2>
+      <p className={text.lap}>124lap</p>
+    </div>
+  );
+};
+
+const containerStyle = flex({
+  gap: '2px',
+  flexDirection: 'column',
+  w: 'full',
+});
```

Comment:
이미 flex pattern 으로 묶은게 명시가 되어있기 때문에 flexDirection -> direction 으로 바꿔도 좋을 것 같아요~~

```suggestion

const containerStyle = flex({
  gap: '2px',
  direction: 'column',
  w: 'full',
});